go/indexer/txmgr/txmgr.go

```diff
@@ -0,0 +1,264 @@
+package txmgr
```

Comment:
We won't need this file at all since we never submit transactions.

Reply:
Done.